src/lib/prover/prover.ml

```diff
@@ -187,8 +204,11 @@ module Worker = struct
     type 'w functions =
       { initialized: ('w, unit, [`Initialized]) F.t
       ; extend_blockchain:
-          ('w, Extend_blockchain_input.t, Blockchain.t Or_error.t) F.t
-      ; verify_blockchain: ('w, Blockchain.t, bool) F.t }
+          ( 'w
```

Comment:
Similar to before; :ype is not versioned, should not talk about versioning modules IMO.